Gordon360/Controllers/LostAndFoundController.cs

```diff
@@ -31,6 +32,20 @@ public ActionResult<int> CreateMissingItemReport([FromBody] MissingItemReportVie
             return Ok(ID);
         }
 
+        /// <summary>
+        /// Update Missing Item Report with the given id with given data
+        /// </summary>
+        /// <param name="id">The id of the report to update</param>
+        /// <returns>ObjectResult - the http status code result of the action</returns>
```

Comment:
The returns comment should also mention that it returns the ID of the action taken with the http status.  See the post request for missing item reports for an example.